src/room.cpp

```diff
@@ -160,7 +160,9 @@ void Room::HandleClientDisconnection(RakNet::AddressOrGUID client) {
 void Room::SendJoinSuccess(const Member& member) {
     RakNet::BitStream stream;
     stream.Write(static_cast<RakNet::MessageID>(ID_ROOM_JOIN_SUCCESS));
-    stream.Write(member.mac_address);
+    for(auto it: member.mac_address) {
```

Comment:
That doesn't really tell me anything, why doesn't it handle endianness correctly if we use the array directly?